pinot-core/src/test/java/org/apache/pinot/queries/DistinctQueriesTest.java

```diff
@@ -319,6 +321,7 @@ private void testDistinctInnerSegmentHelper(String[] queries, boolean isPql)
   @Test
   public void testDistinctInnerSegment()
       throws Exception {
+    //@formatter:off
```

Comment:
You need to have another comment `//@formatter:on` to turn the formatter on after the queries